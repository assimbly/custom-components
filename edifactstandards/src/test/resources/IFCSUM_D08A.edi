UNA:+.? 'UNB+UNOC:3+878542765:01:315000554+0000026335::315904425+201020:0659+22834970513190++IFCSUM'UNH+22834970513190+IFCSUM:D:08A:UN'BGM+343+2010342851+9'DTM+137:202010200658:203'FTX+AAI+++SHIPMENT ORDER PRELEG'FTX+AAI+++Das vorgeschr. Gestellungs-Dat. ist unbedingt einzuhalten Rückfragen bitte an o.g. Tel.Nummer. Ohne unsere Zustimmung ist Deck-, Teil- u. Umladung nicht erlaubt.'FTX+AAI+++Bei Rückfragen und Rechnungen bitte angeben!?: 2010342851'FTX+ADR+++Vorlauf WA VL See LKW'FTX+DCL+++Die Beschaffenheit des Gutes entspricht den Vorschriften des ADN.+DE'FTX+DCL+++The nature of the goods conforms to the provisions of ADN.+EN'CNT+7:20585.60:KGM'RFF+AIW:2010342851'RFF+ACE:YE07'NAD+CZ+63182:160:86++BASF Polyurethanes GmbH+Elastogranstr. 60+Lemförde+03::86:Niedersachsen+49448+DE'CTA+MS+GBC/GX-E1:Frau Rammlau'COM+?+49 5443 12 2505:TE'COM+JANET.RAMMLAU@BASF.COM:EM'TDT+10++3:LKW+31'DTM+181:20201022:102'DTM+181:130000:402'LOC+80+DELME:::Paul Schockemöhle Logistics Damme G Südring 20 49401 Damme+:::0001 PACK'TDT+20++1+13++++9732333:146:11:UMM QARN'DTM+180:20201026:102'LOC+9+DEHAM:::Hamburg'RFF+SI:2010340923'EQD+CN+995370655+42GP:::40´ Mehrzw.Cont. 8´6"'EQN+1'TMD+3:Merchant Haulage E++2'NAD+EP+463053:160:86++Internationale Container Transport:ICT+Sperberweg 6a+Neuss+05::86:Nordrhein-Westfalen+41468+DE'NAD+PW+DE2D0001PACK:160:86++Paul Schockemöhle Logistics Damme G:Dienstleister+Südring 20+Damme++49401+DE'NAD+DCP+463053:160:86++Internationale Container Transport:ICT+Sperberweg 6a+Neuss+05::86:Nordrhein-Westfalen+41468+DE'FTX+AAF+++Leercontainer frei Terminal Seehafen'CNI+1+2030915507'GID+1+20:CP1:67:91:Palette,Holz,1200x1000x138mm,CP1,HT+800:5H4:::Säcke aus Kunststofffolie'PIA+5+50453951:SA'PIA+5+39095090:HS'FTX+AAA+++ESP 9406 25KG 5H4 V'FTX+DEL+++ONE SHIPMENT ONE B/L:USCI#91310116736666737U:Final destination?: Shanghai:Attn?: Ms XU Ping:Tel?: 021-61436722*8017'MEA+WT+AAD+KGM:20125.600'MEA+WT+AAC+KGM:20000.000'MEA+WT+WT+KGM:460.000'MEA+DEN+TC:::Stoffdichte (20°C)+23:1.27'RFF+LI:2030915507:000010'DGS+ZZZ'FTX+AAN+++Geeignete Umschließungswerkstoffe?: Papier/Pappe, Polyethylen hoher Dichte (HDPE), Polyethylen niedriger Dichte (LDPE)'FTX+ACB+++Geruch?: geruchlos'UNT+46+22834970513190'UNZ+1+22834970513190'