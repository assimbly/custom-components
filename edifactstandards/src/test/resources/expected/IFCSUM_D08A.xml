<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blujaysolutions.com/2020/IFCSUM file:///F:/users/Projecten/1.%20Projectdocumentatie/EDIFACTS/IFCSUM%20D08A/IFCSUM_D08A.xsd" xmlns="http://www.blujaysolutions.com/2020/IFCSUM">

    <Interchange>

        <UNB>

            <F0020>22834970513190</F0020>

            <F0026>IFCSUM</F0026>

            <S001>

                <F0001>UNOC</F0001>

                <F0002>3</F0002>

            </S001>

            <S002>

                <F0004>878542765</F0004>

                <F0007>01</F0007>

                <F0008>315000554</F0008>

            </S002>

            <S003>

                <F0010>0000026335</F0010>

                <F0014>315904425</F0014>

            </S003>

            <S004>

                <F0017>201020</F0017>

                <F0019>659</F0019>

            </S004>

        </UNB>

        <UNZ>

            <F0036>1</F0036>

            <F0020>22834970513190</F0020>

        </UNZ>

        <Group>

            <Message_IFCSUM>

                <UNH>

                    <F0062>22834970513190</F0062>

                    <S009>

                        <F0065>IFCSUM</F0065>

                        <F0052>D</F0052>

                        <F0054>08A</F0054>

                        <F0051>UN</F0051>

                    </S009>

                </UNH>

                <BGM>

                    <F1225>9</F1225>

                    <C002>

                        <F1001>343</F1001>

                    </C002>

                    <C106>

                        <F1004>2010342851</F1004>

                    </C106>

                </BGM>

                <DTM>

                    <C507>

                        <F2005>137</F2005>

                        <F2380>202010200658</F2380>

                        <F2379>203</F2379>

                    </C507>

                </DTM>

                <FTX>

                    <F4451>AAI</F4451>

                    <C108>

                        <F4440>SHIPMENT ORDER PRELEG</F4440>

                    </C108>

                </FTX>

                <FTX>

                    <F4451>AAI</F4451>

                    <C108>

                        <F4440>Das vorgeschr. Gestellungs-Dat. ist unbedingt einzuhalten RÃ¼ckfragen bitte an o.g. Tel.Nummer. Ohne unsere Zustimmung ist Deck-, Teil- u. Umladung nicht erlaubt.</F4440>

                    </C108>

                </FTX>

                <FTX>

                    <F4451>AAI</F4451>

                    <C108>

                        <F4440>Bei RÃ¼ckfragen und Rechnungen bitte angeben!: 2010342851</F4440>

                    </C108>

                </FTX>

                <FTX>

                    <F4451>ADR</F4451>

                    <C108>

                        <F4440>Vorlauf WA VL See LKW</F4440>

                    </C108>

                </FTX>

                <FTX>

                    <F4451>DCL</F4451>

                    <F3453>DE</F3453>

                    <C108>

                        <F4440>Die Beschaffenheit des Gutes entspricht den Vorschriften des ADN.</F4440>

                    </C108>

                </FTX>

                <FTX>

                    <F4451>DCL</F4451>

                    <F3453>EN</F3453>

                    <C108>

                        <F4440>The nature of the goods conforms to the provisions of ADN.</F4440>

                    </C108>

                </FTX>

                <CNT>

                    <C270>

                        <F6069>7</F6069>

                        <F6066>20585.6</F6066>

                        <F6411>KGM</F6411>

                    </C270>

                </CNT>

                <UNT>

                    <F0074>46</F0074>

                    <F0062>22834970513190</F0062>

                </UNT>

                <SG1>

                    <RFF>

                        <C506>

                            <F1153>AIW</F1153>

                            <F1154>2010342851</F1154>

                        </C506>

                    </RFF>

                </SG1>

                <SG1>

                    <RFF>

                        <C506>

                            <F1153>ACE</F1153>

                            <F1154>YE07</F1154>

                        </C506>

                    </RFF>

                </SG1>

                <SG4>

                    <NAD>

                        <F3035>CZ</F3035>

                        <F3164>LemfÃ¶rde</F3164>

                        <F3251>49448</F3251>

                        <F3207>DE</F3207>

                        <C082>

                            <F3039>63182</F3039>

                            <F1131>160</F1131>

                            <F3055>86</F3055>

                        </C082>

                        <C080>

                            <F3036>BASF Polyurethanes GmbH</F3036>

                        </C080>

                        <C059>

                            <F3042>Elastogranstr. 60</F3042>

                        </C059>

                        <C819>

                            <F3229>03</F3229>

                            <F3055>86</F3055>

                            <F3228>Niedersachsen</F3228>

                        </C819>

                    </NAD>

                    <SG5>

                        <CTA>

                            <F3139>MS</F3139>

                            <C056>

                                <F3413>GBC/GX-E1</F3413>

                                <F3412>Frau Rammlau</F3412>

                            </C056>

                        </CTA>

                        <COM>

                            <C076>

                                <F3148>+49 5443 12 2505</F3148>

                                <F3155>TE</F3155>

                            </C076>

                        </COM>

                        <COM>

                            <C076>

                                <F3148>JANET.RAMMLAU@BASF.COM</F3148>

                                <F3155>EM</F3155>

                            </C076>

                        </COM>

                    </SG5>

                </SG4>

                <SG9>

                    <TDT>

                        <F8051>10</F8051>

                        <C220>

                            <F8067>3</F8067>

                            <F8066>LKW</F8066>

                        </C220>

                        <C001>

                            <F8179>31</F8179>

                        </C001>

                    </TDT>

                    <DTM>

                        <C507>

                            <F2005>181</F2005>

                            <F2380>20201022</F2380>

                            <F2379>102</F2379>

                        </C507>

                    </DTM>

                    <DTM>

                        <C507>

                            <F2005>181</F2005>

                            <F2380>130000</F2380>

                            <F2379>402</F2379>

                        </C507>

                    </DTM>

                    <SG10>

                        <LOC>

                            <F3227>80</F3227>

                            <C517>

                                <F3225>DELME</F3225>

                                <F3224>Paul SchockemÃ¶hle Logistics Damme G SÃ¼dring 20 49401 Damme</F3224>

                            </C517>

                            <C519>

                                <F3222>0001 PACK</F3222>

                            </C519>

                        </LOC>

                    </SG10>

                </SG9>

                <SG9>

                    <TDT>

                        <F8051>20</F8051>

                        <C220>

                            <F8067>1</F8067>

                        </C220>

                        <C001>

                            <F8179>13</F8179>

                        </C001>

                        <C222>

                            <F8213>9732333</F8213>

                            <F1131>146</F1131>

                            <F3055>11</F3055>

                            <F8212>UMM QARN</F8212>

                        </C222>

                    </TDT>

                    <DTM>

                        <C507>

                            <F2005>180</F2005>

                            <F2380>20201026</F2380>

                            <F2379>102</F2379>

                        </C507>

                    </DTM>

                    <SG10>

                        <LOC>

                            <F3227>9</F3227>

                            <C517>

                                <F3225>DEHAM</F3225>

                                <F3224>Hamburg</F3224>

                            </C517>

                        </LOC>

                    </SG10>

                    <SG16>

                        <RFF>

                            <C506>

                                <F1153>SI</F1153>

                                <F1154>2010340923</F1154>

                            </C506>

                        </RFF>

                    </SG16>

                </SG9>

                <SG22>

                    <EQD>

                        <F8053>CN</F8053>

                        <C237>

                            <F8260>995370655</F8260>

                        </C237>

                        <C224>

                            <F8155>42GP</F8155>

                            <F8154>40Â´ Mehrzw.Cont. 8Â´6"</F8154>

                        </C224>

                    </EQD>

                    <EQN>

                        <C523>

                            <F6350>1</F6350>

                        </C523>

                    </EQN>

                    <TMD>

                        <F8341>2</F8341>

                        <C219>

                            <F8335>3</F8335>

                            <F8334>Merchant Haulage E</F8334>

                        </C219>

                    </TMD>

                    <NAD>

                        <F3035>EP</F3035>

                        <F3164>Neuss</F3164>

                        <F3251>41468</F3251>

                        <F3207>DE</F3207>

                        <C082>

                            <F3039>463053</F3039>

                            <F1131>160</F1131>

                            <F3055>86</F3055>

                        </C082>

                        <C080>

                            <F3036>Internationale Container Transport</F3036>

                            <F3036>ICT</F3036>

                        </C080>

                        <C059>

                            <F3042>Sperberweg 6a</F3042>

                        </C059>

                        <C819>

                            <F3229>05</F3229>

                            <F3055>86</F3055>

                            <F3228>Nordrhein-Westfalen</F3228>

                        </C819>

                    </NAD>

                    <NAD>

                        <F3035>PW</F3035>

                        <F3164>Damme</F3164>

                        <F3251>49401</F3251>

                        <F3207>DE</F3207>

                        <C082>

                            <F3039>DE2D0001PACK</F3039>

                            <F1131>160</F1131>

                            <F3055>86</F3055>

                        </C082>

                        <C080>

                            <F3036>Paul SchockemÃ¶hle Logistics Damme G</F3036>

                            <F3036>Dienstleister</F3036>

                        </C080>

                        <C059>

                            <F3042>SÃ¼dring 20</F3042>

                        </C059>

                    </NAD>

                    <NAD>

                        <F3035>DCP</F3035>

                        <F3164>Neuss</F3164>

                        <F3251>41468</F3251>

                        <F3207>DE</F3207>

                        <C082>

                            <F3039>463053</F3039>

                            <F1131>160</F1131>

                            <F3055>86</F3055>

                        </C082>

                        <C080>

                            <F3036>Internationale Container Transport</F3036>

                            <F3036>ICT</F3036>

                        </C080>

                        <C059>

                            <F3042>Sperberweg 6a</F3042>

                        </C059>

                        <C819>

                            <F3229>05</F3229>

                            <F3055>86</F3055>

                            <F3228>Nordrhein-Westfalen</F3228>

                        </C819>

                    </NAD>

                    <FTX>

                        <F4451>AAF</F4451>

                        <C108>

                            <F4440>Leercontainer frei Terminal Seehafen</F4440>

                        </C108>

                    </FTX>

                </SG22>

                <SG26>

                    <CNI>

                        <F1490>1</F1490>

                        <C503>

                            <F1004>2030915507</F1004>

                        </C503>

                    </CNI>

                    <SG51>

                        <GID>

                            <F1496>1</F1496>

                            <C213>

                                <F7224>20</F7224>

                                <F7065>CP1</F7065>

                                <F1131>67</F1131>

                                <F3055>91</F3055>

                                <F7064>Palette,Holz,1200x1000x138mm,CP1,HT</F7064>

                            </C213>

                            <C213>

                                <F7224>800</F7224>

                                <F7065>5H4</F7065>

                                <F7064>SÃ¤cke aus Kunststofffolie</F7064>

                            </C213>

                        </GID>

                        <PIA>

                            <F4347>5</F4347>

                            <C212>

                                <F7140>50453951</F7140>

                                <F7143>SA</F7143>

                            </C212>

                        </PIA>

                        <PIA>

                            <F4347>5</F4347>

                            <C212>

                                <F7140>39095090</F7140>

                                <F7143>HS</F7143>

                            </C212>

                        </PIA>

                        <FTX>

                            <F4451>AAA</F4451>

                            <C108>

                                <F4440>ESP 9406 25KG 5H4 V</F4440>

                            </C108>

                        </FTX>

                        <FTX>

                            <F4451>DEL</F4451>

                            <C108>

                                <F4440>ONE SHIPMENT ONE B/L</F4440>

                                <F4440>USCI#91310116736666737U</F4440>

                                <F4440>Final destination: Shanghai</F4440>

                                <F4440>Attn: Ms XU Ping</F4440>

                                <F4440>Tel: 021-61436722*8017</F4440>

                            </C108>

                        </FTX>

                        <SG53>

                            <MEA>

                                <F6311>WT</F6311>

                                <C502>

                                    <F6313>AAD</F6313>

                                </C502>

                                <C174>

                                    <F6411>KGM</F6411>

                                    <F6314>20125.600</F6314>

                                </C174>

                            </MEA>

                        </SG53>

                        <SG53>

                            <MEA>

                                <F6311>WT</F6311>

                                <C502>

                                    <F6313>AAC</F6313>

                                </C502>

                                <C174>

                                    <F6411>KGM</F6411>

                                    <F6314>20000.000</F6314>

                                </C174>

                            </MEA>

                        </SG53>

                        <SG53>

                            <MEA>

                                <F6311>WT</F6311>

                                <C502>

                                    <F6313>WT</F6313>

                                </C502>

                                <C174>

                                    <F6411>KGM</F6411>

                                    <F6314>460.000</F6314>

                                </C174>

                            </MEA>

                        </SG53>

                        <SG53>

                            <MEA>

                                <F6311>DEN</F6311>

                                <C502>

                                    <F6313>TC</F6313>

                                    <F6154>Stoffdichte (20Â°C)</F6154>

                                </C502>

                                <C174>

                                    <F6411>23</F6411>

                                    <F6314>1.27</F6314>

                                </C174>

                            </MEA>

                        </SG53>

                        <SG55>

                            <RFF>

                                <C506>

                                    <F1153>LI</F1153>

                                    <F1154>2030915507</F1154>

                                    <F1156>000010</F1156>

                                </C506>

                            </RFF>

                        </SG55>

                        <SG66>

                            <DGS>

                                <F8273>ZZZ</F8273>

                            </DGS>

                            <FTX>

                                <F4451>AAN</F4451>

                                <C108>

                                    <F4440>Geeignete UmschlieÃ&#159;ungswerkstoffe: Papier/Pappe, Polyethylen hoher Dichte (HDPE), Polyethylen niedriger Dichte (LDPE)</F4440>

                                </C108>

                            </FTX>

                            <FTX>

                                <F4451>ACB</F4451>

                                <C108>

                                    <F4440>Geruch: geruchlos</F4440>

                                </C108>

                            </FTX>

                        </SG66>

                    </SG51>

                </SG26>

            </Message_IFCSUM>

        </Group>

    </Interchange>

</Envelope>